include/deal.II/base/config.h.in

```diff
@@ -40,8 +40,6 @@
 #cmakedefine DEAL_II_WITH_ASSIMP
 #cmakedefine DEAL_II_WITH_COMPLEX_VALUES
 #cmakedefine DEAL_II_WITH_CUDA
-#cmakedefine DEAL_II_WITH_CXX14
-#cmakedefine DEAL_II_WITH_CXX17
```

Comment:
What do we do with the define flag for `CXX17` that a user code might depend on? You have added a backwards compatibility setting for `..._WITH_CXX14` below, but not for `_WITH_CXX17`. Am I missing something?

Reply:
I have added the `DEAL_II_WITH_CXX17` variables back to both configuration places (deal.IIConfig.cmake, config.h).